bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/console/ChannelConsoleCommandExtension.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.internal.console;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.smarthome.core.events.EventPublisher;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.events.ThingEventFactory;
+import org.eclipse.smarthome.io.console.Console;
+import org.eclipse.smarthome.io.console.extensions.AbstractConsoleCommandExtension;
+
+/**
+ * Console command extension to access channels
+ *
+ * @author Stefan Triller - Initial contribution
+ *
+ */
+public class ChannelConsoleCommandExtension extends AbstractConsoleCommandExtension {
```

Comment:
I also briefly discussed this with kai this morning and he said that the only useful (non-debugging) feature   for channels is to trigger it. Thats why I thought I rename the class to `TriggerChannelConsoleCommandExtension`, since I only use it for triggering an event on a channel and there will not be any further subcommand in the future.